soh/include/z64save.h

```diff
@@ -284,7 +284,7 @@ typedef struct {
     // #endregion
     // #region SOH [Randomizer]
     // Upstream TODO: Move these to their own struct or name to more obviously specific to Randomizer
-    /*        */ u16 randomizerInf[17];
+    /*        */ u16 randomizerInf[RAND_INF_MAX / 16];
```

Comment:
As it is right now the last flags get written oob (if there was 20 flags => 20 / 16 = 1 => only space for 16).
```suggestion
    /*        */ u16 randomizerInf[(RAND_INF_MAX + 15) / 16];
```